modules/programs/rofi.nix

```diff
@@ -410,6 +410,15 @@ in {
           Cannot use the rofi options 'theme' and 'colors' simultaneously.
         '';
       }
+      (hm.assertions.assertPlatform "programs.rofi" pkgs platforms.linux)
+      {
+        assertion = (!builtins.hasAttr "override" cfg.package)
```

Comment:
AFAIK everything `callPackage`d is also overridable so the first `builtins.hasAttr` check is not useful.

Reply:
`pkgs.writeScriptBin`s used in some tests are not overridable, tests will crash without this check.

Reply:
Makes sense.

It would also prevent people (unless they used makeOverridable) from using simple wrapper scripts.

I did not think about this enough. :+1: